tests/Unit/Handlers/Eloquent/Schema/migrations/default_users_table_anon/2014_10_12_000000_create_users_table.php

```diff
@@ -7,12 +7,10 @@
 return new class extends Migration {
     /**
      * Run the migrations.
-     *
-     * @return void
      */
-    public function up()
+    public function up(): void
```

Comment:
let's keep these files in their original versions as they were created by Laravel, and our goal is to test files generated by Laravel (I mean all migrations)

Reply:
It's the default since Laravel 10.
https://github.com/laravel/laravel/blob/10.x/database/migrations/2014_10_12_000000_create_users_table.php

Reply:
thank you, I didn't know that